runtime/jcl/exports.cmake

```diff
@@ -701,259 +701,124 @@ if(J9VM_OPT_JFR)
 			Java_jdk_jfr_internal_JVM_getPid
 			Java_jdk_jfr_internal_JVM_getStackTraceId
 			Java_jdk_jfr_internal_JVM_getThreadId
 			Java_jdk_jfr_internal_JVM_getTicksFrequency
 			Java_jdk_jfr_internal_JVM_getTimeConversionFactor
-			Java_jdk_jfr_internal_JVM_getTypeId__Ljava_lang_Class_2
-			Java_jdk_jfr_internal_JVM_getTypeId__Ljava_lang_String_2
 			Java_jdk_jfr_internal_JVM_getUnloadedEventClassCount
-			Java_jdk_jfr_internal_JVM_hostTotalMemory
-			Java_jdk_jfr_internal_JVM_hostTotalSwapMemory
-			Java_jdk_jfr_internal_JVM_include
 			Java_jdk_jfr_internal_JVM_isAvailable
-			Java_jdk_jfr_internal_JVM_isContainerized
-			Java_jdk_jfr_internal_JVM_isExcluded__Ljava_lang_Class_2
-			Java_jdk_jfr_internal_JVM_isExcluded__Ljava_lang_Thread_2
-			Java_jdk_jfr_internal_JVM_isInstrumented
-			Java_jdk_jfr_internal_JVM_isRecording
 			Java_jdk_jfr_internal_JVM_log
-			Java_jdk_jfr_internal_JVM_logEvent
-			Java_jdk_jfr_internal_JVM_markChunkFinal
-			Java_jdk_jfr_internal_JVM_nanosNow
 			Java_jdk_jfr_internal_JVM_newEventWriter
 			Java_jdk_jfr_internal_JVM_registerNatives
-			Java_jdk_jfr_internal_JVM_registerStackFilter
 			Java_jdk_jfr_internal_JVM_retransformClasses
 			Java_jdk_jfr_internal_JVM_setCompressedIntegers
-			Java_jdk_jfr_internal_JVM_setConfiguration
 			Java_jdk_jfr_internal_JVM_setCutoff
-			Java_jdk_jfr_internal_JVM_setDumpPath
 			Java_jdk_jfr_internal_JVM_setEnabled
 			Java_jdk_jfr_internal_JVM_setFileNotification
 			Java_jdk_jfr_internal_JVM_setForceInstrumentation
 			Java_jdk_jfr_internal_JVM_setGlobalBufferCount
 			Java_jdk_jfr_internal_JVM_setGlobalBufferSize
 			Java_jdk_jfr_internal_JVM_setMemorySize
-			Java_jdk_jfr_internal_JVM_setMethodSamplingPeriod
-			Java_jdk_jfr_internal_JVM_setMiscellaneous
 			Java_jdk_jfr_internal_JVM_setOutput
 			Java_jdk_jfr_internal_JVM_setRepositoryLocation
 			Java_jdk_jfr_internal_JVM_setStackDepth
 			Java_jdk_jfr_internal_JVM_setStackTraceEnabled
 			Java_jdk_jfr_internal_JVM_setThreadBufferSize
 			Java_jdk_jfr_internal_JVM_setThreshold
-			Java_jdk_jfr_internal_JVM_setThrottle
 			Java_jdk_jfr_internal_JVM_shouldRotateDisk
 			Java_jdk_jfr_internal_JVM_storeMetadataDescriptor
 			Java_jdk_jfr_internal_JVM_subscribeLogLevel
 			Java_jdk_jfr_internal_JVM_uncaughtException
-			Java_jdk_jfr_internal_JVM_unregisterStackFilter
 		)
+
+		if(NOT JAVA_SPEC_VERSION LESS 17)
+			# Java 17+
+			omr_add_exports(jclse
+				Java_jdk_jfr_internal_JVM_emitDataLoss
+				Java_jdk_jfr_internal_JVM_exclude
+				Java_jdk_jfr_internal_JVM_flush__
+				Java_jdk_jfr_internal_JVM_getChunkStartNanos
+				Java_jdk_jfr_internal_JVM_getTypeId__Ljava_lang_Class_2
+				Java_jdk_jfr_internal_JVM_getTypeId__Ljava_lang_String_2
+				Java_jdk_jfr_internal_JVM_include
+				Java_jdk_jfr_internal_JVM_isRecording
+				Java_jdk_jfr_internal_JVM_logEvent
+				Java_jdk_jfr_internal_JVM_markChunkFinal
+				Java_jdk_jfr_internal_JVM_setMethodSamplingPeriod
+				Java_jdk_jfr_internal_JVM_setThrottle
+			)
```

Comment:
Please don't nest versions like this. Instead group entries by version following this pattern:
```
	if(JAVA_SPEC_VERSION EQUAL 11)
		# Java 11 only
		...
	endif()
	if(NOT JAVA_SPEC_VERSION LESS 11)
		# Java 11+
		...
	endif()

	if(JAVA_SPEC_VERSION EQUAL 17)
		# Java 17 only
		...
	endif()
	if(NOT JAVA_SPEC_VERSION LESS 17)
		# Java 17+
		...
	endif()
	...
```

Reply:
Removed nest versions.